src/k8s_autoscale/validate.py

```diff
@@ -6,3 +6,17 @@ def validate(config, schema):
     config = yaml.safe_load(config)
     schema = yaml.safe_load(schema)
     jsonschema.validate(config, schema)
+    verify_slo_task_duration(config)
+
+
+def verify_slo_task_duration(config):
+    def verify_slo(worker_type):
+        if (
+            worker_type["autoscale"]["args"]["slo_seconds"]
+            < worker_type["autoscale"]["args"]["avg_task_duration"]
+        ):
+            return f"{worker_type['worker_type']}: slo_seconds shouldn't be less than avg_task_duration"
```

Comment:
How long is the interval from resolving one task to claiming the next, and does that impact on the SLO measurement?

Reply:
It should be close to 0. At least we don't have any logic in the scriptworker code to sleep between the claimWork calls IFAIK.